kbase/narrative
# Dataup 213 clear errors from staging

# Description of PR purpose/changes

This PR creates a `clear all` button when a file upload to the dropzone errors. The clear all button will remove all errored files from the dropzone when clicked. It will not remove files that are still uploading. 

I talked to Mallory about the button positioning and she is fine with it's location given the scrolling functionality when there is more than a single errored file in the dropzone.

![Screen Shot 2020-10-16 at 1 00 37 PM](https://user-images.githubusercontent.com/56279459/96308368-baa4fb80-0faf-11eb-9a2c-64d4d62b0d51.png)


# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-213

# Testing Instructions
* Details for how to test the PR: 
To execute the unit tests, navigate to the narrative repo and enter `make test-frontend-unit`

To view this change locally, complete the following steps:
* Pull this branch
* Execute `kbase-narrative'
* Create a new narrative
* Navigate to the data import panel
* Drag and drop a file that is greater than 20 GB into the dropzone. This should cause a file too large error.
* Repeat to get a group of files in the dropzone that have errored
* click the `CLEAR ALL` button
This should clear the errored files from the drop zone.

# Dev Checklist:


# Updating Version and Release Notes (if applicable)
